lib/middleware/user-data/user-data.js

```diff
@@ -59,8 +57,26 @@ const dataStoreRemote = {
   *
   * @return {undefined}
   */
-  loadData: async (args, logger) => {
-    const data = await userDataStoreClient.getData(args, logger)
+  async loadData (args, logger) {
+    let data = {}
+    try {
+      data = await userDataStoreClient.getData(args, logger)
+    } catch (e) {
+      const {
+        data: {
+          response: {
+            statusCode
+          } = {}
+        } = {},
+        message
+      } = e
+
+      const code = Number(statusCode)
+
+      error(code, message)
+
+      if (code !== 404) throw e

```

Comment:
This is _not doing anything differently_ except saying, "hey, error from the `client` library, do you have a `statusCode` field buried down on the attributes of your error? You _do_? Excellent -- let's just make sure that's a number and the number is `404`. If it _isn't_ I'm going to re-throw your error"